### DISCUSSION TITLE
RSA SHA-1 Error

### DISCUSSION THREAD

**Discussion by babucarr32** [community/community#37726] [Other Feature Feedback, Questions, & Ideas]
Hello guys how can I solve this error **ERROR: You're using an RSA key with SHA-1, which is no longer allowed. Please use a newer client or a different key type.** in git? I see this error when I do **git push -u origin main**, Thank you.

**Comment by airtower-luna** [comment]
That sounds like the SSH client you're using is extremely outdated. Make sure to update.

**Reply by Kaiosn** [reply]
microsoft account is not working what to do with it
